CONTRIBUTING.md

```diff
@@ -178,6 +178,14 @@ following rules before submitting:
    as your Python, scikit-learn, numpy, and scipy versions. This information
    can be found by running the following code snippet:
 
+   For scikit-learn >= 0.20:
+
+  ```python
+  import sklearn; sklearn.show_versions()
+  ```
+
+   For scikit-learn < 0.20:
```

Comment:
The `For scikit-learn < 0.20` I think that have a fail indentation. 

![image](https://user-images.githubusercontent.com/7605307/46290573-9873e880-c562-11e8-8a0f-720ab7613c17.png)

Reply:
thanks @eamanu for the review